app/models/service_orchestration.rb

```diff
@@ -25,8 +25,9 @@ def orchestration_stack_status
   end
 
   def deploy_orchestration_stack
+    creation_options = stack_options
```

Comment:
@Ladas this will evaluate the new manager and template before passing them to the create method.